src/components/dialog/Dialog.vue

```diff
@@ -132,7 +132,13 @@ export default defineComponent({
     background-color: rgba(0, 0, 0, 0.4);
     display: flex;
     justify-content: center;
+    align-items: flex-start;
     transition: opacity 0.3s ease;
+    overflow: scroll;
+
+    @media (min-width: 430px) {
+      align-items: center;
+    }
   }
```

Comment:
this change was implemented as a fix for the overlay not scrolling in smaller screens

Reply:
To make this work properly you should set `.dialog__mask` to have overflow: auto; and align-items: flex-start; (auto is preferred since scroll forces scrollbars on windows regardless of overflowing) and `.dialog__wrapper` to have margin-top: auto; margin-bottom: auto;

This will center the dialog on big screens, align it to the top of the screen on mobile if the content is taller than the screen, and also only show scrollbars when needed